components/navbar/sample.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/css/bootstrap-reboot.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT Sans">
+    <link rel="stylesheet" href="style.css">
+    <title>Sample Navigation</title>
+</head>
+
+<!--
+  All in-line CSS is specific to this sample; it can and should be ignored.
+ -->
+
+<body style="background: #F8F8F8; margin-top: 80px;">
+  <header>
+    <div class="banner">
+        <a href="https://gov.bc.ca" alt="British Columbia">
+          <img src="../assets/images/logo-banner.png" alt="logo" />
+        </a>
+        <h1>Hello British Columbia</h1>
+    </div>
+    <div class="other">
+      <a class="nav-btn">
+        <i class="fas fa-bars"></i>
+      </a>
+    <!-- 
+      This place is for anything that needs to be right aligned
+      beside the logo.
+    -->
+    </div>
+    </div>
```

Comment:
extra div

Reply:
is there going to be a seperate component for the `<nav>..</nav>`

Reply:
Fixed. Also, I'm going to add the nav to this component next.